sensing/pointcloud_preprocessor/src/passthrough_filter/passthrough_filter_uint16_nodelet.cpp

```diff
@@ -94,25 +94,27 @@ rcl_interfaces::msg::SetParametersResult PassThroughFilterUInt16Component::param
   }
 
   // Check the current value for keep_organized
-  bool keep_organized;
-  get_param(p, "keep_organized", keep_organized);
-  if (impl_.getKeepOrganized() != keep_organized) {
-    RCLCPP_DEBUG(
-      get_logger(), "Setting the filter keep_organized value to: %s.",
-      keep_organized ? "true" : "false");
-    // Call the virtual method in the child
-    impl_.setKeepOrganized(keep_organized);
+  bool keep_organized{};
+  if (get_param(p, "keep_organized", keep_organized)) {
```

Comment:
Why did you add this `if`? I guess if `get_param` fails `keep_organized` will be `false`.

Reply:
The failure of `get_param` means that parameter is not updated, so set parameter function of `impl_` should be blocked by this `if` condition.

Reply:
Simplify at https://github.com/autowarefoundation/autoware.universe/pull/198/commits/faaa97a3544affaf15cb6cf4f342c645293c43ef.